help/md/dnCBDSP.md

```diff
@@ -1,9 +1,38 @@
 ## name
 dnCBDSP
 ## title
+Conditional birth-death shift process
 ## description
+Simulates a tree under a birth-death process with shifts in birth and death rates.
 ## details
+This function simulates a tree under a birth-death process. The initial 
+birth and death rates can be specified with the rootAge and rootLambda arguments.
+The rates at which speciation and extinction shifts is specified by the delta
+argument and the new speciation or extinction rate can be drawn from a prior distribution
+specfied in the lambda and mu arguments. 
+Similar to other birth-death processes in RevBayes, you can specify the stopping condition
+of the simulator (either survival or time) and the sampling probability.
 ## authors
 ## see_also
+dnCDBDP
+dnSBBDP
 ## example
+# set distributions for tree
+root_age ~ dnUniform(0, 2)
+root_lambda ~ dnUniform(0, 1)
+root_mu ~ dnUniform(0, 1)
+sampling_prob := 1/2
```

Comment:
`sampling_prob := 1/2` – I can't see why this variable should be deterministic; its value doesn't depend on any other variable.